package.json

```diff
@@ -73,15 +93,17 @@
     "preprocess": "run-p preprocess:*",
     "preprocess:svg-to-twig": "gulp svg-to-twig",
     "preprocess:theo-to-mdx": "gulp theo-to-mdx",
-    "watch-preprocess": "gulp watch",
+    "watch-preprocess": "gulp watch-preprocess",
+    "build": "gulp build",
     "check-lint": "run-s check-lint:*",
-    "check-lint:prettier": "prettier --ignore-path .gitignore --list-different '**/*.{scss,js,mdx}'",
+    "check-lint:prettier": "prettier --ignore-path .gitignore --check '**/*.{scss,js,mdx}'",
```

Comment:
This change is minor and somewhat unrelated, I just noticed that last year Prettier added the `--check` option which is similar to `--list-different` but it prints useful output like "prettier is checking that these files match formatting" ... "These files did not match prettier, did you forget to run the formatter"